tests/testthat/test-post-tweet.R

```diff
@@ -53,3 +45,12 @@ test_that("post_tweet geolocated works", {
   rt <- expect_message(post_destroy(crt$id_str), "your tweet has been deleted!")
   expect_equal(httr::status_code(rt), 200L)
 })
+
+test_that("post_tweet geolocated validity checks", {
+  msg <- paste("test geolocated error", Sys.time()) # To avoid having duplicated status
+  expect_snapshot(post_tweet(msg, lat = "x", long = 0), error = TRUE)
```

Comment:
For this type of snapshotting, I'd put them all in one snapshot, like:

```R
expect_snapshot(error = TRUE, {
  ...
})
```

Reply:
Changed to the proposed approach